extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/extractors/PeertubeAccountExtractor.java

```diff
@@ -55,8 +58,29 @@ public String getFeedUrl() throws ParsingException {
     }
 
     @Override
-    public long getSubscriberCount() {
-        return json.getLong("followersCount");
+    public long getSubscriberCount() throws ParsingException {
+        long subscribersCount = json.getLong("followersCount");
+        String accountVideoChannelUrl = baseUrl + PeertubeChannelLinkHandlerFactory.API_ENDPOINT;
+        if (getId().contains(ACCOUNTS)) {
+            accountVideoChannelUrl += getId();
+        } else {
+            accountVideoChannelUrl += ACCOUNTS + getId();
+        }
+        accountVideoChannelUrl += "/video-channels";
+
+        try {
+            final String responseBody = getDownloader().get(accountVideoChannelUrl).responseBody();
+            final JsonObject jsonResponse = JsonParser.object().from(responseBody);
+            final JsonArray videoChannels = jsonResponse.getArray("data");
+            for (final Object videoChannel : videoChannels) {
+                final JsonObject videoChannelJsonObject = (JsonObject) videoChannel;
+                subscribersCount += videoChannelJsonObject.getInt("followersCount");
+            }
+        } catch (final IOException | JsonParserException | ReCaptchaException ignored) {
+            // something went wrong during video channels extraction, only return subscribers of ownerAccount
+        }
+        System.out.println(subscribersCount);
```

Comment:
Remove this

Reply:
I was thinking I remove all prints but no, sorry. Will fix it later.